### PR TITLE
Add support to install missing packages on running system

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: test/bootloader_base_test.rb [hunk 1]
```diff
@@ -14,6 +19,14 @@
 
       subject.write
     end
+
+    context "Mode.normal is set" do
+      it "install packages listed required by bootloader from packages method" do
```
Comment (lslezak) [inline test/bootloader_base_test.rb]: Hardly understandable to me, probably simple `installs packages required by bootloader` would be enough.

Comment (jreidinger) [inline test/bootloader_base_test.rb]: ok, make sense

**Comment by lslezak** [comment]
LGTM

**Comment by jreidinger** [comment]
thanks

**Pull request merged**